src/cr/cube/matrix.py

```diff
@@ -1469,14 +1451,14 @@ def __init__(
         table_margin,
         zscore=None,
         column_index=None,
-        opposite_margins=None,
+        **kwargs
```

Comment:
Using `**kwargs` seems unnecessary here. What's wrong with `opposite_margins=None`? It accomplishes the same thing and is more explicit.

Reply:
I used `kwargs` because I would propagate opposit margins to be used in `_addend_vectors` like here https://github.com/Crunch-io/crunch-cube/blob/a135ca6d1cf9ff55c28fad4a07f10a0744d6086b/src/cr/cube/matrix.py#L994

Reply:
What is different about this behavior (using `**kwargs`) and just having `opposite_margins=None` in there like you did before? The `x=y` argument syntax makes the argument optional, so adding the `kwargs` layer just obscures what is a simple and common pattern of an optional parameter.

Reply:
mmmm U totally right, I don't remenber why that param was `None` when I define that new property. I'll change it in the next commit